LetsDinnerV2 MessagesExtension/Model/Event.swift

```diff
@@ -366,7 +366,25 @@ class Event {
                 
                 if let ingredients = dict[DataKeys.ingredients] as? [String : String] {
                     ingredients.forEach { key, value in
-                        let customIngredient = LDIngredient(name: key + value)
```

Comment:
Do not delete this line, comment it out, the solution below is temporary as you said